libs/src/services/creatorNode/index.js

```diff
@@ -344,12 +344,14 @@ class CreatorNode {
   /**
    * Uploads an image to the connected content node
    * @param {File} file image to upload
-   * @param {boolean?} square whether this image should be turned into a square (e.g. profile picture / track artwork)
+   * @param {string} route route to handle upload (image_upload, track_upload, etc.)
```

Comment:
this previous jsdoc comment is still relevant?

Reply:
except with the needed extra timeoutMs comment

Reply:
good catch done